src/resources/views/crud/fields/relationship/fetch.blade.php

```diff
@@ -191,6 +186,7 @@ function bpFieldInitFetchElement(element) {
         var $allows_null = element.attr('data-column-nullable') == 'true' ? true : false;
         var $appLang = element.attr('data-app-current-lang');
         var $selectedOptions = JSON.parse(element.attr('data-selected-options') ?? null);
+        var $multiple = (element.attr('data-field-multiple') == 'true') ? true : false;
```

Comment:
See above, I think it's better to check if the item has `multiple` on it, rather than `data-field-multiple`. Single source of truth.